Source/SelfService/Web/microservice/microserviceView/configuration/AlertDialog.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import * as React from 'react';
+
+import { Button } from '@dolittle/design-system/atoms/Button/Button';
+
+import { IconButton, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+import { CloseRounded } from '@mui/icons-material';
+
+export const AlertDialog = ({ open, onClose, handleDeletionConfirm }: any) =>
```

Comment:
If this dialog component is mean to be only for deleting a microservice then it can be renamed to `DeleteMicroserviceDialog?